webui/react/src/components/ColumnPickerMenu.tsx

```diff
@@ -21,6 +22,8 @@ const BANNED_COLUMNS: Set<string> = new Set([]);
 const removeBannedColumns = (columns: ProjectColumn[]) =>
   columns.filter((col) => !BANNED_COLUMNS.has(col.column));
 
+export const METADATA_SEPARATOR = '\u241F' as const;
```

Comment:
can we move this [here](https://github.com/determined-ai/determined/pull/10066/files#diff-b3f8c84db4390025a5db99526f13b9320f76cd069d83a295ab3c7ec59810d3b5)